discovery-provider/src/api/v1/users.py

```diff
@@ -1096,6 +1097,14 @@ def get(self, id):
     "users_by_content_node", full_ns, fields.List(fields.Nested(user_replica_set))
 )
 
+users_by_content_node_response = make_full_response(
```

Comment:
`users_by_content_node_response` is already defined above.

maybe move `user_replica_set_response` close to where it's used